Microsoft-graph-toolkit v1.1 documentation update (#6425)
microsoftgraph/microsoft-graph-docs: concepts/toolkit/templates.md

> **Note:** You can also expand objects such as `{{event}}` and they will render as JSON strings. This can be useful when you're developing the templates.

> **Context**

Reviewer: ```suggestion
## Data Context helper properties
```

Author: [updated title](https://github.com/microsoftgraph/microsoft-graph-docs/pull/6425/files#diff-880fdc26a24a1a1eebafb18a328632a6R66)

> There are also accessible properties when binding, for template context. 

Reviewer: ```suggestion
The following properties can also be used with the data context object in your templates
```

Author: [updated](https://github.com/microsoftgraph/microsoft-graph-docs/pull/6425/commits/826c791e0854be1b0dfc11dcc6147413c40b9d8d#diff-880fdc26a24a1a1eebafb18a328632a6R67-R68)

| Property |  Description |
| --- | --- | --- |
| `$index` | numerical index of item being rendered while being looped with `data-for` |

> | `$parent` | If a template is rendered inside of another template, the user is able to find data context of the parent within the child template. |

Reviewer: ```suggestion
| `$parent` | If a template is rendered inside of another template, the `$parent` property allows the developer to access the parent data context. |
```

Author: [updated context description](https://github.com/microsoftgraph/microsoft-graph-docs/pull/6425/commits/826c791e0854be1b0dfc11dcc6147413c40b9d8d#diff-880fdc26a24a1a1eebafb18a328632a6R73)

```html

> <mgt-person>

Reviewer: This example doesn't really make sense. You can use this example:

```html
<span data-for="language in languages">
    {{ language.displayName }}<span data-if="$index < languages.length - 1">, </span>
</span>
```

Author: [Updated example](https://github.com/microsoftgraph/microsoft-graph-docs/pull/6425/commits/826c791e0854be1b0dfc11dcc6147413c40b9d8d#diff-880fdc26a24a1a1eebafb18a328632a6R79-R81)
